README.md

```diff
@@ -5,9 +5,11 @@
 
```

Comment:
haha thanks! i'm having a lot of fun working on this. yes, i plan to keep working on this; i plan on adding a few features slowly in the future including:
- multiple shipping options (not just overnight)
- graphic interface of some sort to make it easier for user to understand what is happening, if there are unexpected errors
 
thank you so much for offering to let me be a project "owner" -- i'll make sure to only push fully stable code to this repo, and leave my beta repo to be the playground haha